crates/rolldown/tests/esbuild/default/export_wildcard_fs_node_common_js/artifacts.snap

```diff
@@ -22,11 +21,11 @@ Object.keys(fs).forEach(function (k) {
   });
 });
 
-var ___external = require("./external");
-Object.keys(___external).forEach(function (k) {
+var external = require("./external");
+Object.keys(external).forEach(function (k) {
```

Comment:
Here the relative external module `resolved id` is a absolute path, the `name` is a relative path, the `name` is empty if the external module is `.`, found the case at rollup test `rollup@function@external-directory-import: handles using ../ as external import`. 
So here using the normalized relative path to generate the name, if the external module is '.', after the normalied it is '.'. Using relative path from root, because here common dir is not correctly, ref https://github.com/rolldown/rolldown/pull/3834#discussion_r1993066008/